packages/devtools_app/lib/src/screens/performance/panes/timeline_events/perfetto/_perfetto_web.dart

```diff
@@ -53,7 +57,10 @@ class _PerfettoState extends State<Perfetto> with AutoDisposeMixin {
   }
 
   void _loadActiveTrace() {
-    _viewController._loadTrace(_perfettoController.activeTraceEvents.value);
+    assert(_perfettoController.activeTraceEvents.value != null);
+    _viewController._loadTrace(
+      _perfettoController.activeTraceEvents.value ?? [],
+    );
```

Comment:
Why don't we bang instead of assert here?

With assert, if null squeeze somehow, feature will silently stop working properly and we will never find this out.

Another option is to function gracefully and issue analytic event.

Reply:
used band instead.